std-bits/base/src/main/java/org/enso/base/enso_cloud/EnsoPathFileSystemSPI.java

```diff
@@ -0,0 +1,21 @@
+package org.enso.base.enso_cloud;
+
+import org.enso.base.file_system.FileSystemSPI;
+
+@org.openide.util.lookup.ServiceProvider(service = FileSystemSPI.class)
+public class EnsoPathFileSystemSPI extends FileSystemSPI {
+  @Override
+  protected String getModuleName() {
+    return "Standard.Base.Enso_Cloud.Enso_File";
+  }
+
+  @Override
+  protected String getTypeName() {
+    return "Enso_File";
+  }
+
+  @Override
+  protected String getProtocol() {
+    return "enso";
```

Comment:
Where is the specification of this protocol? Shouldn't the behavior be defined somewhere in this PR?

Reply:
We don't really have a proper specification but it's just `enso://<org_name>/<path>`.

I added a short summary in f4944e1a176686b688097141ec276a8275aa46f9